stdlib/test/builtin/test_string.mojo

```diff
@@ -664,8 +721,19 @@ fn test_lstrip() raises:
     var str2 = String(" \t\n\t\v\fsomething")
     assert_true(str2.lstrip() == "something")
 
+    # with custom chars for lstrip
+    var str3 = String("mississippi")
+    assert_true(str3.lstrip("mis") == "ppi")
+
+    var str4 = String(" \n mississippimississippi")
+    assert_true(str4.lstrip("mis ") == "\n mississippimississippi")
+    assert_true(str4.lstrip("mis \n") == "ppimississippi")
+
 
 fn test_strip() raises:
+    print("start strip")
```

Comment:
**Suggestion** Remove this `print`.  It's not needed for anything.

Reply:
Done :)